routers/comments.js

```diff
@@ -0,0 +1,89 @@
+const express = require('express')
+let db = require('../data/db')
+
+const router = express.Router({
+    mergeParams: true
+})
+
+// 	_______CRUD_________
+
+
+//______________________________________________________________
+// get - (http://localhost:4000/api/posts/:id/comments)
+router.get('/', (req, res) => {
+    const id = req.params.id;
+
+    db.findPostComments(id)
+        .then(data => {
```

Comment:
be careful how you're handling the data that is returned.  the `findPostComments` returns an array regardless of if that particular post actually exists.  Because of that you won't ever get into your catch statement.  You need to do a  `findById` first and then nest the `findPostComments` inside the initial then statement